test/unit/utils/bundle-utils.spec.js

```diff
@@ -16,7 +16,7 @@ describe('BundleUtils.bundleResource', () => {
         expect(fs.existsSync(PathUtils.calculateAbsolutePath('test/unit/fixtures/bundled.js'))).to.be.true
         done()
       })
-  })
+  }).timeout(5000)
```

Comment:
How about `this.timeout(5000)` in the function body (we have 2 other examples like that).  I think that form is clearer to read.

Reply:
Hmm, for me `it(...).timeout(5000)` is more readable because allows to separate from test content and test configuration. We are also able to use arrow function in that case. 

What about changing other places into that style ? 

Although it's detail, so I'm okay with `this.timeout()` as well

Reply:
Ok the arrow function argument convinced me ;-)

Reply:
Nice :)